packages/react-components/react-carousel/stories/src/Carousel/CarouselTopNavigation.stories.tsx

```diff
@@ -0,0 +1,164 @@
+import { Button, CarouselSlider, Image, makeStyles, tokens, typographyStyles } from '@fluentui/react-components';
+import {
+  Carousel,
+  CarouselAnnouncerFunction,
+  CarouselCard,
+  CarouselNav,
+  CarouselNavButton,
+  CarouselNavContainer,
+  CarouselViewport,
+  Text,
+} from '@fluentui/react-components';
+import * as React from 'react';
+
+const useClasses = makeStyles({
+  bannerCard: {
+    alignContent: 'center',
+    borderRadius: tokens.borderRadiusXLarge,
+    boxShadow: tokens.shadow16,
+    height: '450px',
+    textAlign: 'left',
+    position: 'relative',
+  },
+  image: {
+    borderRadius: 'inherit',
+  },
+  cardContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '8px',
+
+    position: 'absolute',
+    left: '10%',
+    top: '25%',
+    borderRadius: tokens.borderRadiusLarge,
+    boxShadow: tokens.shadow8,
+    background: tokens.colorNeutralBackground1,
+    padding: '24px 32px',
+    maxWidth: '270px',
+    width: '50%',
+  },
+  title: {
+    ...typographyStyles.title3,
+  },
+  subtext: {
+    marginBottom: '12px',
+    ...typographyStyles.body1,
+  },
+  container: {
+    display: 'grid',
+    gridTemplateColumns: '1fr',
+    gridTemplateRows: 'auto 1fr',
+  },
+  card: {
+    minHeight: '100px',
+  },
+  carousel: {
+    flex: 1,
+  },
+  controls: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '6px',
+
+    padding: '10px',
+  },
+  field: {
+    flex: 1,
+    gridTemplateColumns: 'minmax(100px, max-content) 1fr',
+  },
+  dropdown: {
+    maxWidth: 'max-content',
+  },
+  carouselHeader: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    gap: '10px',
+    marginBottom: '16px',
+  },
+  carouselHeaderTitle: { flex: '1', margin: '0', fontSize: '22px', fontWeight: 800 },
```

Comment:
We should not use non-Fluent typography. Please pick a matching preset instead: https://storybooks.fluentui.dev/react/?path=/docs/components-text--docs#presets